modules/ROOT/pages/_partials/data-sync/js-client-mutate.adoc

```diff
@@ -2,7 +2,7 @@
 
 .Prerequisites
 * You have Node.js and npm installed.
-* You have completed link:{data-sync-queries-link}[Queries section] and the server is still running.
+* You have completed the link:{data-sync-queries-link}[Queries section] and the server is still running.
```

Comment:
Not sure if 'completed' or 'finished' is the right word here but it doesn't really matter I suppose.